modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl

```diff
@@ -129,6 +130,14 @@ public abstract class Base${schemaName}ResourceTestCase {
 		testCompany = CompanyLocalServiceUtil.getCompany(testGroup.getCompanyId());
 
 		<#if generateDepotEntry>
+			irrelevantDepotEntry = DepotEntryLocalServiceUtil.addDepotEntry(
```

Comment:
I don't love the irrelevant tests in the collections... I am considering removing them in the future because it's a test executing 2 things (this code is inside a conditional), behind an obscure condition. Ideally I would extract it to another test that checks that unrelated results don't appear in the collection. Just an observation for the future.

Reply:
Removed to stop adding more logic as it will be removed in the future and complicates other parts of the tests